pkg/reconciler/knativeeventing/common/images_test.go

```diff
@@ -172,6 +172,96 @@ var updateDeploymentImageTests = []updateDeploymentImageTest{
 			Env:   []corev1.EnvVar{{Name: "SOME_IMAGE", Value: "docker.io/my/overridden-image"}},
 		}},
 	},
+	{
```

Comment:
It would be nice to test a few more corner cases:

* container X in multiple deployments
* Specify just X-Y, doesn't apply to container called X-Y